web/src/state_data.ts

```diff
@@ -141,6 +142,7 @@ export const realm_schema = z.object({
     realm_logo_source: z.string(),
     realm_logo_url: z.string(),
     realm_move_messages_between_streams_policy: z.number(),
+    realm_mandatory_topics: z.boolean(),
```

Comment:
Just a small comment.
`realm_mandatory_topics` will come before `realm_move_messages_between_streams_policy` alphabetically :)

Reply:
Thanks for kind review. I have made the requested change 😃